api/licenses/serializers.py

```diff
@@ -11,7 +11,14 @@ class LicenseSerializer(JSONAPISerializer):
         'name',
         'id',
     ])
-    non_anonymized_fields = ['type']
+    anonymized_fields = [
+        'id',
```

Comment:
Same here -- if there's no `id`, ember data chokes.

Why do we need to anonymize licenses? The personal information is stored on the node.

Reply:
Not sure, we were anonymizing them previously, this work doesn't change that.  I agree with you that there's not much they would get from knowing the license info, but maybe we should do some historical digging or check with product.

Reply:
Looks like this is actually a difference between legacy and APIv2 -- on legacy, licenses are perfectly visible and can actually expose contributor info:
![Screen Shot 2019-05-07 at 10 45 20](https://user-images.githubusercontent.com/6776190/57309121-87212a80-70b5-11e9-8962-97c371785c71.png)
https://test.osf.io/q94tg/?view_only=89c33ef552144e18a8007a9ce4c81643

So, perfect feature parity would be both `LicenseSerializer` and `NodeSerializer.node_license` non-anonymized. Since that seems buggy, we should either hide the license altogether or just mask the node-specific `node_license` bit. I'll consult with product about which seems better.